internal/view/js/component/dialog.js

```diff
@@ -109,27 +109,31 @@ export default {
 		fields: {
 			immediate: true,
 			handler() {
-				this.formFields = this.fields.map(field => {
-					if (typeof field === 'string') return {
+				this.formFields = this.fields.reduce((fields, field) => {
```

Comment:
Why this change? Is this related to the PR?

Reply:
Yes, it's because `if (field.visible === false) return` in a `map` insert a null in the array and Vue don't like this. If my memory is good, it's the reason why I replaced the `map` by a `reduce`, to filter null.

So why there seemed to not have problem with `if (typeof field === 'string') return` before? Maybe `typeof field` is always equal to string but I have not verified I admit.

Reply:
Because I add a new checkbox to force the caching only if the caching previously failed. If the caching worked great, no need to force it so no need to display it I think.

However, it's all the time displayed is multiple bookmarks are selected.